src/components/Home/Home.js

```diff
@@ -1,24 +1,47 @@
-import { useTheme } from '@react-navigation/native';
-import React from 'react';
-import { Text, View } from 'react-native';
-import { useSelector } from 'react-redux';
+import React, { useEffect, useState } from 'react';
+import { View, Image, FlatList } from 'react-native';
+import useFetchMovies from '../../hooks/useFetchMovies';
```

Comment:
Prefer using the babel "shortcuts" instead of `../..` check babel.config.js to see or add new aliases